pages/api/Schema/reports/twelveMonth/datahandler.ts

```diff
@@ -112,7 +112,11 @@ export const mapTwelveMonthReport = (
             return {
               id: contactDonationInfo.contact_id,
               name: contact?.contact_name ?? '',
-              total: Number(contactDonationInfo.total),
+              total: contactDonationInfo.months.reduce(
```

Comment:
Can you walk me through why this needed changed? If the report contained 12 months of $100, was the total not $1200?